.tests/integration/config.yaml

```
@@ -26,6 +26,20 @@ cnvkit_scatter:
 cnvkit_vcf:
   container: "docker://hydragenetics/cnvkit:0.9.9"
 
+cnvkit_seg:
+  container: "docker://hydragenetics/cnvkit:0.9.9"
+
+purecn:
+  container: "docker://markusriester/purecn:2.2.0"
+  genome: "hg19"
+  segmentation_method: "GATK4"
+  fun_segmentation: "Hclust"
+  interval_padding: 100
+
+purecn_coverage:
+  container: "docker://markusriester/purecn:2.2.0"
+  intervals: "/reference/targets_intervals_25000.txt"
```

is this file missing? I can't find it in the pull-request and should it be without the starting slash?

```suggestion
  intervals: "reference/targets_intervals_25000.txt"
```